client/call.go

```diff
@@ -39,3 +40,133 @@ func (c *Call) resetClientCmdID(clock Clock) {
 		Random:   rand.Int63(),
 	}
 }
+
+// Method returns the name of the database command for the call.
+func (c *Call) Method() string {
+	return c.Args.Method()
+}
+
+// GetCall returns a Call object initialized to get the value at key.
+func GetCall(key proto.Key, reply *proto.GetResponse) *Call {
```

Comment:
It's not much extra work to fix. This entire change has been done incrementally. This was just a step that a) I wasn't sure if it needed doing and b) I hadn't gotten to yet. Removing the `reply` argument can easily be done in a separate change.